src/main/java/com/company/controller/CallController.java

```diff
@@ -115,4 +122,26 @@ public void doCall(HttpServletRequest request, HttpServletResponse response) thr
         }
     }
 
+
+    @GetMapping("/callEnd")
+    public void callEnd(HttpServletRequest request, HttpServletResponse response){
+
+        callService.writeCallToDB(request);
+
+        VoiceResponse.Builder responseBuilder = new VoiceResponse.Builder();
+        VoiceResponse twiml = responseBuilder.build();
+
+        response.setContentType(CAll_CONTENT_TYPE);
+
+        try {
+            response.getWriter().print(twiml.toXml());
```

Comment:
1. response.getWriter().print(twiml.toXml()); - it's response on callEnd API. i.e. you returned xml on client. 
well, if you'll get a error, then client ... what? 
in the end, if will a error, then client'll fail of 504 (https://httpstatuses.com/504) 
2. why logger.warn? I'm seeing that be logger.error. If should be "warn", then should info text "why is it warn?".
https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels
3. try-catch - for all method use.